Update aioharmony to 0.1.13

ehendrix23 opened the pull request:
## Description:
Increase version of aioharmony to 0.1.13 

**Related issue (if applicable):** fixes #24889 

## Checklist:
  - [X] The code change is tested and works locally.
  - [X] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [X] There is no commented out code in this PR.
  - [X] I have followed the [development checklist][dev-checklist]

If the code communicates with devices, web services, or third-party tools:
  - [X] [_The manifest file_][manifest-docs] has all fields filled out correctly. Update and include derived files by running `python3 -m script.hassfest`.
  - [X] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
  - [] Untested files have been added to `.coveragerc`.

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

Commit:
Update aioharmony to 0.1.13

Increase version of aioharmony to 0.1.13

homeassistant commented:
Hello @ehendrix23,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`bed078db861e47ef8797fe7185b0bcdb129a05e3`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant&#x2F;pull&#x2F;26399&#x2F;commits&#x2F;bed078db861e47ef8797fe7185b0bcdb129a05e3) - This commit has something that looks like an email address (ehendrix@macc02yh4zpjhd2.amgen.com). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

Commit:
Update aioharmony to 0.1.13

Increase version of aioharmony to 0.1.13

Committer: Erik Hendrix <hendrix_erik@hotmail.com>

Commit:
Update aioharmony to 0.1.13

Increase version of aioharmony to 0.1.13

Committer: Erik Hendrix <hendrix_erik@hotmail.com>

ehendrix23 committed:
Merge branch 'dev' of https://github.com/ehendrix23/home-assistant into dev

homeassistant commented:
Hello @ehendrix23,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`3430e72f3e9975c851480a1d56e1baf734e9eca1`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant&#x2F;pull&#x2F;26399&#x2F;commits&#x2F;3430e72f3e9975c851480a1d56e1baf734e9eca1) - This commit has something that looks like an email address (ehendrix@macc02yh4zpjhd2.amgen.com). Maybe try linking that to GitHub?.
- [`bed078db861e47ef8797fe7185b0bcdb129a05e3`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant&#x2F;pull&#x2F;26399&#x2F;commits&#x2F;bed078db861e47ef8797fe7185b0bcdb129a05e3) - This commit has something that looks like an email address (ehendrix@macc02yh4zpjhd2.amgen.com). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:

pvizeli committed:
Update OpenCV 4.1.1 / Numpy 1.17.1 (#26387)

frenck committed:
Use literal string interpolation in integrations T-W (f-strings) (#26394)

frenck committed:
Use literal string interpolation in integrations K-M (f-strings) (#26389)

frenck committed:
Use literal string interpolation in integrations R-S (f-strings) (#26392)

frenck committed:
Use literal string interpolation in integrations X-Z (f-strings) (#26395)

Commit:
Merge branch 'dev' of https://github.com/ehendrix23/home-assistant into dev

homeassistant commented:
Hello @ehendrix23,

When attempting to inspect the commits of your pull request for CLA signature status among all authors we encountered commit(s) which were not linked to a GitHub account, thus not allowing us to determine their status(es).

The commits that are missing a linked GitHub account are the following:

- [`eb2b35cf6ae4f70a3f524fcd08f11f08f2b680fb`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant&#x2F;pull&#x2F;26399&#x2F;commits&#x2F;eb2b35cf6ae4f70a3f524fcd08f11f08f2b680fb) - This commit has something that looks like an email address (hendrix_erk@hotmail.com). Maybe try linking that to GitHub?.
- [`81ba9e4b9d787e33a862dc37ffc73089e945a37e`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant&#x2F;pull&#x2F;26399&#x2F;commits&#x2F;81ba9e4b9d787e33a862dc37ffc73089e945a37e) - This commit has something that looks like an email address (ehendrix@macc02yh4zpjhd2.amgen.com). Maybe try linking that to GitHub?.
- [`3430e72f3e9975c851480a1d56e1baf734e9eca1`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant&#x2F;pull&#x2F;26399&#x2F;commits&#x2F;3430e72f3e9975c851480a1d56e1baf734e9eca1) - This commit has something that looks like an email address (ehendrix@macc02yh4zpjhd2.amgen.com). Maybe try linking that to GitHub?.
- [`bed078db861e47ef8797fe7185b0bcdb129a05e3`](https:&#x2F;&#x2F;github.com&#x2F;home-assistant&#x2F;home-assistant&#x2F;pull&#x2F;26399&#x2F;commits&#x2F;bed078db861e47ef8797fe7185b0bcdb129a05e3) - This commit has something that looks like an email address (ehendrix@macc02yh4zpjhd2.amgen.com). Maybe try linking that to GitHub?.

Unfortunately, **we are unable to accept this pull request until this situation is corrected.**

Here are your options:

1. If you had an email address set for the commit that simply wasn't linked to your GitHub account you can link that email now and it will retroactively apply to your commits. The simplest way to do this is to click the link to one of the above commits and look for a blue question mark in a blue circle in the top left. Hovering over that bubble will show you what email address you used. Clicking on that button will take you to your email address settings on GitHub. Just add the email address on that page and you're all set. GitHub has more information about this option [in their help center](https://help.github.com/articles/setting-your-email-in-git/#commits-on-github-arent-linking-to-my-account).

2. If you didn't use an email address at all, it was an invalid email, or it's one you can't link to your GitHub, you will need to change the authorship information of the commit and your global Git settings so this doesn't happen again going forward. GitHub provides some great instructions on how to change your authorship information [in their help center](https://help.github.com/articles/setting-your-email-in-git/).
   * If you only made a single commit you should be able to run
      ```
      git commit --amend --author="Author Name <email@address.com>"
      ```
      (substituting Author Name and email@address.com for your actual information) to set the authorship information.
   * If you made more than one commit and the commit with the missing authorship information is not the most recent one you have two options:
       1. You can re-create all commits missing authorship information. This is going to be the easiest solution for developers that aren't extremely confident in their Git and command line skills.
       2. You can use [this script](https://help.github.com/articles/changing-author-info/) that GitHub provides to rewrite history. **Please note:** this should be used only if you are very confident in your abilities and understand its impacts.
   * Whichever method you choose, I will come by to re-check the pull request once you push the fixes to this branch.

We apologize for this inconvenience, especially since it usually bites new contributors to Home Assistant. We hope you understand the need for us to protect ourselves and the great community we all have built legally. The best thing to come out of this is that you only need to fix this once and it benefits the entire Home Assistant and GitHub community.

Thanks, I look forward to checking this PR again soon! :heart:
